src/core/seq.js

```
@@ -0,0 +1,124 @@
+// 'seq' command: generate numeric sequences
+// 'seq' command: generate numeric sequences with support for -f, -s, -w flags
+function main({ args = [], stdin = '', env = {} }) {
+  // parse flags
+  let fmt = null;
+  let sep = '\n';
+  let equalWidth = false;
+  let i = 0;
+  function stripQuotes(s) {
+    if (s.length > 1) {
+      const a = s[0], b = s[s.length - 1];
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)